osmo_jupyter/plot.py

```diff
@@ -168,19 +188,39 @@ def get_rgb_columns(df, pattern, other_columns_to_include=None):
     '''
     other_columns_to_include = other_columns_to_include or []
 
-    color_columns = [
-        pattern.replace('*', color)
-        for color in COLOR_CHANNELS
-    ]
+    colors_with_pattern = _get_colors_with_pattern(pattern)
 
-    all_desired_columns = color_columns + other_columns_to_include
+    all_desired_columns = colors_with_pattern + other_columns_to_include
     rename_map = {
         original_column: new_column
-        for original_column, new_column in zip(color_columns, COLOR_CHANNELS)
+        for original_column, new_column in zip(colors_with_pattern, COLOR_CHANNELS)
     }
     return df[all_desired_columns].rename(columns=rename_map)
 
 
+def add_rgb_columns(df_to_modify, rgb_df, pattern):
+    ''' Add rgb columns to a DataFrame (IN PLACE!) with a provided prefix or suffix pattern
+
+    eg.
+    >>> main_df = pd.DataFrame(..., columns=['Timestamp', ...])
+    >>> rgb_df = pd.DataFrame(..., columns=['r', 'g', 'b'])
```

Comment:
I don't get why this df is in this example

Reply:
ohh. Is this the actual means? Maybe the name could be clearer then.

```
>>> rgb_means_df = pd.DataFrame(..., columns=['r', 'g', 'b'])
```

Reply:
done